idapython.cpp

```diff
@@ -1129,8 +1129,7 @@ ref_t idapython_plugin_t::get_sys_displayhook()
   ref_t h;
   if ( config.repl_use_sys_displayhook )
   {
-    ref_t py_sys(PyW_TryImportModule("sys"));
-    if ( py_sys != nullptr )
+    if ( ref_t py_sys = ref_t(PyW_TryImportModule("sys")) )
```

Comment:
Unless I'm mistaken, this implies C++-17 (conditional with initializer). 
Should we really impose that? (it's minor syntactic sugar)

Also, this doesn't seem to be harmonized across the entire patch (see the previous chunk)